src/dropdown/dropdown.component.ts

```diff
@@ -166,6 +172,10 @@ export class Dropdown implements OnInit, AfterContentInit, OnDestroy {
 	 * Defaults to the `DROPDOWN.SELECTED` value from the i18n service.
 	 */
 	@Input() selectedLabel = this.i18n.get().DROPDOWN.SELECTED;
+	/**
+	 * Template to render the current value of the dropdown.
+	 */
+	@Input() renderTpl: string | TemplateRef<any> = null;
```

Comment:
We should be able to re-use `displayValue` instead of introducing another Input ... we'll just need to check if `displayValue` contains a template or not and toggle the template appropriately